From 75681d5c3cf09a3b31ddf517af24c957951ea970 Mon Sep 17 00:00:00 2001
From: Brent Cook <busterb@gmail.com>
Date: Mon, 20 Apr 2026 06:35:57 -0500
Subject: [PATCH 21/21] clock_gettime is in time.h

---
 src/usr.sbin/ntpd/ntp_dns.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/usr.sbin/ntpd/ntp_dns.c b/src/usr.sbin/ntpd/ntp_dns.c
index 7cd97fe1baf..d256e8ab75d 100644
--- a/src/usr.sbin/ntpd/ntp_dns.c
+++ b/src/usr.sbin/ntpd/ntp_dns.c
@@ -31,6 +31,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <syslog.h>
+#include <time.h>
 #include <unistd.h>
 
 #include "ntpd.h"
-- 
2.53.0

